assets/src/edit-story/components/canvas/useCanvasKeys.js

```diff
@@ -63,25 +63,41 @@ function useCanvasKeys(ref) {
     }
   );
 
-  // Return focus back to the canvas when another section loses the focus.
+  /**
+   * Sets default focus node from body to the ref element.
+   */
   useEffect(() => {
     const container = ref.current;
     if (!container) {
       return undefined;
     }
-
+    /**
+     * If node loses focus, wait to see if another node gains it.
+     * if not focus default element.
+     */
     const doc = container.ownerDocument;
-
     const handler = () => {
-      // Make sure that no other component is trying to get the focus
-      // at this time. We have to check all "focusout" events here because
-      // after DOM removal, the "focusout" events are all over the place.
+      /**
+       * setTimeout places callback in the back of the IO queue after
+       * any postential focusin events are queued.
+       */
       setTimeout(() => {
         if (doc.activeElement === doc.body) {
           container.focus();
         }
-      }, 300);
+      }, 0);
```

Comment:
This should accomplish the same thing putting this behind any `focusin` events in the IO queue. Happy to change it back tho as it doesn't really change much for the user

Reply:
I'd be in favor of risking it. But we should get a QA pass on this since this has plagued us for some time.